plugins/catalog/src/components/EntityContextMenu/EntityContextMenu.tsx

```diff
@@ -144,6 +164,24 @@ export function EntityContextMenu(props: EntityContextMenuProps) {
             </ListItemIcon>
             <ListItemText primary="Inspect entity" />
           </MenuItem>
+          <MenuItem
+            onClick={() => {
+              copyToClipboard();
+            }}
+          >
+            <ListItemIcon>
+              <FileCopyTwoToneIcon fontSize="small" />
+            </ListItemIcon>
+            <ListItemText primary="Copy entity url" />
+          </MenuItem>
+          <Snackbar
```

Comment:
Oh by the way. You should probably use the alert API for this, not rendering it yourself.

```ts
const alertApi = useApi(alertApiRef);
alertApi.post({ message: 'Copied!', severity: 'info' });
```

Reply:
Ah! ya that's much better and simpler :)